DonjinKrawler-Common/src/main/java/krawlercommon/enemies/Enemy.java

```diff
@@ -95,15 +150,19 @@ public String getInfo() {
         return info;
     }
 
-    public void showOnScreen() {
-        System.out.println(this.getName() + " is on screen.");
+    public void move() {
+        if ((dx < 0 && x - dx > 2) || (dx > 0 && x + 20 + dx < 500))
```

Comment:
I presume this allows enemies to move through walls. If so, file a bug report so we remember to fix that afterwards.